pandas/tests/tseries/offsets/test_offsets_properties.py

```diff
@@ -97,40 +95,39 @@ def test_on_offset_implementations(dt, offset):
     assert offset.is_on_offset(dt) == (compare == dt)
 
 
-@pytest.mark.xfail(
-    reason="res_v2 below is incorrect, needs to use the "
-    "commented-out version with tz_localize.  "
-    "But with that fix in place, hypothesis then "
-    "has errors in timezone generation."
-)
 @given(gen_yqm_offset, gen_date_range)
 def test_apply_index_implementations(offset, rng):
     # offset.apply_index(dti)[i] should match dti[i] + offset
-    assume(offset.n != 0)  # TODO: test for that case separately
 
-    # rng = pd.date_range(start='1/1/2000', periods=100000, freq='T')
+    # TODO: test for that case separately
+    assume(offset.n != 0)
+
     ser = pd.Series(rng)
 
     res = rng + offset
     res_v2 = offset.apply_index(rng)
-    # res_v2 = offset.apply_index(rng.tz_localize(None)).tz_localize(rng.tz)
+    res_v2 = offset.apply_index(rng.tz_localize(None)).tz_localize(rng.tz)
     assert (res == res_v2).all()
 
+    # apply_index is only for indexes, not series, so no res2_v2
     assert res[0] == rng[0] + offset
     assert res[-1] == rng[-1] + offset
     res2 = ser + offset
-    # apply_index is only for indexes, not series, so no res2_v2
+
+    # TODO: Check randomly assorted entries, not just first/last
     assert res2.iloc[0] == ser.iloc[0] + offset
     assert res2.iloc[-1] == ser.iloc[-1] + offset
-    # TODO: Check randomly assorted entries, not just first/last
 
 
 @pytest.mark.xfail  # TODO: reason?
 @given(gen_yqm_offset)
 def test_shift_across_dst(offset):
```

Comment:
I couldn't remove the ```xfail``` for this test case, @jbrockmendel mind taking a look?

Reply:
@MomIsBestFriend I asked you to take a look, no backsies

Reply:
so no luck here?

Reply:
> so no luck here?

No :disappointed: I could not make this stable/work properly

Reply:
I appreciate you taking a look.  With enough eyeballs we'll eventually get this figured out.